Update CR request for VCDM v2.0. (#55)
w3c/verifiable-credentials: admin/CR-vcdm/Approval_request.md

(Draft for a CR transition of VCDM v2.0 and VC Specifications Directory)

# Document title, URLs, estimated publication date

- Verifiable Credentials Data Model v2.0

>   - https://w3c.github.io/vc-data-model/CR/2023-12-21/

Reviewer: ```suggestion
  - https://www.w3.org/TR/2023/CR-vc-data-model-2.0-20231221/
```

  - 21 December 2023
  - Editors' draft: https://w3c.github.io/vc-data-model/

# Abstract

> - https://w3c.github.io/vc-data-model/CR/2023-12-21/#abstract

Reviewer: ```suggestion
- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-20231221/#abstract
```

# Status

> - https://w3c.github.io/vc-data-model/CR/2023-12-21/#sotd

Reviewer: ```suggestion
- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-20231221/#sotd
```

# Link to group's decision to request transition

- https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-12-@^@#resolution@^^@

# Changes

> - https://w3c.github.io/vc-data-model/CR/2023-12-21/#revision-history

Reviewer: ```suggestion
- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-20231221/#revision-history
```

# Requirements satisfied

Yes.

# Dependencies met (or not)

All normative dependencies are either RECs or IETF RFCs, except for:

* Verifiable Credential Data Integrity, which entered CR in November 2023.

> * Securing Verifiable Credentials using JOSE and COSE, which is a WD and expected to enter CR in Q1 2024.

Reviewer: In view of https://github.com/w3c/vc-data-model/pull/1338 [VC-SPECS] may also join this list as a 'draft note' (unless we decide to publish it as a final note right away).

Reviewer: The goal is to publish that document as a Note. By the time the VCDM is requested to CR this should be done. This comment is therefore moot.

# Wide Review

Issues processed:
- https://github.com/w3c/vc-data-model/issues?q=is%3Aissue+is%3Aclosed+created%3A%3E2022-06-13

PRs processed:
- https://github.com/w3c/vc-data-model/pulls?q=is%3Aissue+is%3Aclosed+created%3A%3E2022-06-13

Horizontal reviews:

> * TAG: https://github.com/w3ctag/design-reviews/issues/860 (request timeout)

Reviewer: They way I read https://github.com/w3ctag/design-reviews/issues/850#issuecomment-1664352922 it is pretty much looking as an approval; no substantial changes required. Also, I seem to remember that both TAG reviewers were at one of our F2F meetings in Sevilla.

I think it would be worth asking the reviewers to put an official mark into the issue whereby this is simply completed. That would look way better than a "request timeout".

Author: That TAG design review was for VC Data Integrity, not the core data model.

I have asked the reviewers to update the VCDM review request here:

https://github.com/w3ctag/design-reviews/issues/860#issuecomment-1821056346

* PING: https://github.com/w3cping/privacy-request/issues/121 (completed)
* Security: https://github.com/w3c/security-request/issues/58 (request timeout)
* a11y: https://github.com/w3c/a11y-request/issues/65 (completed)
* i18n: https://github.com/w3c/i18n-request/issues/212 (completed)

Liasons:

> * There are participants' overlap with the following groups:

Reviewer 2: ```suggestion
* There is participant overlap with the following groups:
```

Author: Fix grammar in participant overlap.
